experimental/LieAlgebras/test/simple_lie_algebra-test.jl

```diff
@@ -0,0 +1,40 @@
+@testset "LieAlgebras.SimpleLieAlgebra" begin
+  @testset "conformance tests" begin
+    @testset "B2(QQ)" begin
+      L = lie_algebra(QQ, :B, 2)
+      lie_algebra_conformance_test(
+        L, SimpleLieAlgebra{QQFieldElem}, SimpleLieAlgebraElem{QQFieldElem}
+      )
+    end
```

Comment:
Added a conformance test, that test some general properties of all LieAlgebra subtypes (some probabilistically).